md/BonitaBPM_platform_setup.md

```diff
@@ -49,6 +49,17 @@ The script `setup` comes with 4 commands:
   Eg. `setup.sh init`  
   Eg. `setup.sh init -Ddb.vendor=postgres` (see [Advanced use](#advanced_use) for information on using `-D` properties)
 
+  ::: info
+  When `db.vendor` and `bdm.db.vendor` properties are both set to `h2` - which is the default configuration - the `init` command asks you to confirm this choice before continuing. The following message is displayed:
+  ```
+  [WARN] Default H2 configuration detected. This is not recommended for production. If this is not the required configuration, change file 'database.properties' and run again.
+  Are you sure you want to continue? (y/n):
+  ```
+
+  If you want to bypass this confirmation prompt you can add the `h2.noconfirm` property as follows:  
+  `setup.sh init -Dh2.noconfirm`
```

Comment:
This should also be available with start_bonita command:
`./start-bonita.sh -Dh2.noconfirm`

Should we document it?

Reply:
Rephrased documentation to add start-bonita example in commit https://github.com/bonitasoft/bonita-doc/pull/374/commits/6dd5d786687758814f8df1539aba580229f55320